tests/performance/tests/checkout-spike.js

```diff
@@ -0,0 +1,24 @@
+import { cart } from '../requests/shopper/cart.js';
```

Comment:
It would be a good idea to change the name of the file from `checkout-spike.js` to something else since this isn't a spike anymore.

Reply:
Good call, I have removed it for now as it was only there for the spike. Have a separate issue for adding extra types of sample scenarios. 
98-gh-woocommerce/woocommerce-quality/